src/modules/likes.js

```diff
@@ -0,0 +1,19 @@
+export default class Likes {
+  static likeUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/sSOo1SGKw5gz6ax9ywvh/likes';
+
+  static postLike = async (id) => {
+    const res = await fetch(this.likeUrl, {
+      method: 'POST',
+      headers: { 'Content-type': 'application/JSON' },
+      body: JSON.stringify({ item_id: id }),
+    });
+    const like = await res.text();
+    return like;
+  }
+
+  static fetchLike = async () => {
+    const res = await fetch(this.likeUrl);
+    const like = await res.json();
+    return like;
+  }
+}
```

Comment:
- [ ] Kindly look into your like functionality. I notice I can only like items that have likes already but I am not able to like an item that has no existing likes. See the example of some of the items I tried to like but nothing happens in return. Even when I reload the page the likes are not added.

  I think this may have to do with the issue that all your items have duplicates on the page. The ones with likes and the ones with no likes. All the birds appear twice. The duplicates without likes can never be "liked". Kindly remove all duplicate items on the page.

  ![image](https://github.com/otmaneechchafyky/javascript-capstone/assets/54848363/8095ea8a-b2d8-4938-944e-fc176ae76dd4)

Reply:
Okay thank you, I am going to work on it.